video/out/vo.h

```diff
@@ -40,7 +40,7 @@ enum {
     VO_EVENT_ICC_PROFILE_CHANGED        = 1 << 2,
     // Some other window state changed (position, window state, fps)
     VO_EVENT_WIN_STATE                  = 1 << 3,
-    // The ambient light conditions changed and need to be reloaded
+    // Deprecated. The ambient light conditions changed and need to be reloaded
     VO_EVENT_AMBIENT_LIGHTING_CHANGED   = 1 << 4,
```

Comment:
This can be removed too, no? It is not public API.

Reply:
yeah, i wasn't sure how to handle this since we would get a 'hole' in that enum or would we want to renumber it?

Reply:
hole is fine, I think, to keep it simple. There are plenty of bits left for new values.

Reply:
If something new is added later, we can always just fill the hole in too.